pom.xml

```diff
@@ -68,6 +68,16 @@
     </repositories>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-configuration2</artifactId>
+            <version>2.2</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-beanutils</groupId>
+            <artifactId>commons-beanutils</artifactId>
+            <version>1.9.3</version>
+        </dependency>
```

Comment:
@jglick @oleg-nenashev do you have any insight if these dependencies are safe to add?
´commons-beanutils´ is provided by code IIRC?

Reply:
Cannot recall offhand, but should be checked.

Reply:
@bipolar please check

Reply:
Indeed jenkins-core already provides beanutils, I've removed the dependency.